src/Style/Display/TooltipStyle.ts

```diff
@@ -9,17 +9,9 @@ export const TooltipContainer = styled.div`
 export const TooltipContent = styled.div<TooltipContentProps>`
   position: absolute;
   z-index: 10;
-
-  ${({ text }) => {
-    if (text.length > 150) {
-      return `
-        width: ${text.length + 20}px;
-      `;
-    }
-    return `
-      width: 150px;
-    `;
-  }}
+  width: inherit;
+  min-width: 150px;
+  max-width: 225px;
```

Comment:
Oh these are to make sure it is compatible with the previous component. If we don't set it it could mess up the existing Tooltips in our codebases :sweat_smile:

Reply:
Okay on second thought I don't need to set the `max-width` but I still need the `min-width` so that components using Tooltip don't break.

Reply:
@aaa59891 done. PTAL!